web/package/cockpit-d-installer.changes

```diff
@@ -16,6 +22,7 @@ Mon Jan  2 10:03:18 UTC 2023 - David Diaz <dgonzalez@suse.com>
 
 - Ensure custom fonts are including in the build
   (gh#yast/d-installer#385).
+>>>>>>> origin/master
```

Comment:
Some reminder after a merge, please remove it.

Reply:
@lslezak thanks, fixed